packages/amplify-e2e-core/src/index.ts

```diff
@@ -22,10 +22,11 @@ declare global {
 const amplifyTestsDir = 'amplify-e2e-tests';
 
 export function getCLIPath(testingWithLatestCodebase = false) {
-  if (isCI() && !testingWithLatestCodebase) {
-    return process.env.AMPLIFY_PATH || 'amplify';
-  }
-  return path.join(__dirname, '..', '..', 'amplify-cli', 'bin', 'amplify');
+  return testingWithLatestCodebase
```

Comment:
Why do we need to return `amplify-dev` if isCI is false? It's possible that a developer might have the repo pulled in multiple locations and amplify-dev is pointed to a different one (ie I do this sometimes ;). Seems like pointing to the relative bin path is more reliable?